examples/2d/cpu_draw.rs

```diff
@@ -0,0 +1,126 @@
+use bevy::prelude::*;
+use bevy::render::render_resource::Extent3d;
+use bevy::render::render_resource::TextureDimension;
+use bevy::render::render_resource::TextureFormat;
+use rand::Rng;
+
+const IMAGE_WIDTH: u32 = 256;
+const IMAGE_HEIGHT: u32 = 256;
+
+fn main() {
+    App::new()
+        .add_plugins(DefaultPlugins)
+        // Let's make the fixed timestep really fast for this example.
```

Comment:
could you explain why?

Reply:
I wanted to make an example that pokes pixels one at a time, at a fixed rate (not the display frame rate), to draw a spiral pattern. I felt like using bevy's fixed timestep, because it naturally lets me do things at a fixed rate idiomatically, instead of having to roll my own thing using Time. But doing it one pixel at a time makes the pattern emerge very slowly, and it is hard to see what's going on, when you run the example. So I decided to make it really fast. Bevy's fixed update will loop to make sure it runs enough times.

Reply:
I mean, in the example so that it's easier to use as a learning resource